conda/environment.yml

```diff
@@ -44,6 +44,9 @@ dependencies:
   - pip:
     # NOTE: required for event website
     - jinja-markdown==1.210911
+    # Database access repo
+    - git+https://github.com/SnowEx/snowexsql.git
```

Comment:
thanks @micahjohnson150 can you use a specific commit or tag here if there aren't any releases (`git+https://github.com/SnowEx/snowexsql.git@v0.1`)?  See these docs on generating a new lockfile for the environment https://github.com/snowex-hackweek/website2022/blob/main/conda/README.md

Reply:
actually packages not on pypi are a little tricky with our locking setup, here is how we did things recently for icesat2 hackweek https://github.com/ICESAT-2HackWeek/website2022/blob/10229090a94edb3e5734dbc00149b0502ad5396a/conda/lock-environment.sh#L39-L41

Reply:
Hi @scottyhq I took some time to make the package pip install-able (on my list to do anyways). I am attempting to relock the environment file but I am having errors. 


```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/micah/miniconda3/envs/CondaLock/lib/python3.10/site-packages/conda_lock/src_parser/__init__.py", line 285, in seperator_munge_get
    return d[key.replace("-", "_")]
KeyError: 'matplotlib_base'
```

Any ideas?

Reply:
unfortunately we're running into this open issue on conda-lock, so will have to work around it https://github.com/conda-incubator/conda-lock/issues/179#issuecomment-1134470426